src/unix/linux_like/linux/arch/generic/mod.rs

```diff
@@ -37,10 +37,6 @@ pub const SO_PASSCRED: ::c_int = 16;
 pub const SO_PEERCRED: ::c_int = 17;
 pub const SO_RCVLOWAT: ::c_int = 18;
 pub const SO_SNDLOWAT: ::c_int = 19;
-pub const SO_RCVTIMEO: ::c_int = 20;
-pub const SO_SNDTIMEO: ::c_int = 21;
-// pub const SO_RCVTIMEO_OLD: ::c_int = 20;
-// pub const SO_SNDTIMEO_OLD: ::c_int = 21;
```

Comment:
I think maybe we could define both the `_OLD` and `_NEW` values to be consistent with the headers (maybe don't make them public if they're not in the docs?) and then make use of them within the config block.

Reply:
The `_OLD` and `_NEW` variants are not mentioned in `socket(7)` where the other constants are listed, so I added them as non-public.  Some of them are mentioned in `semver/TODO-linux.txt`.